### DISCUSSION TITLE
dumpConfigAtStartup to write config to log file

### DISCUSSION THREAD

**Discussion by vic0824** [ArcadeData/arcadedb#623] [Ideas]
ArcadeDB server writes log messages to a log file (as specified in arcadedb-log.properties), but when dumpConfigAtStartup = true, the config is dumped only to System.out.
If the server is started in embedded mode, the config dump is lost, unless the calling application is started by redirecting stdout to a dedicated log file.
Since ArcadeDB server already writes messages to a log file, it would be useful if the config dump was also written to the log file.

**Comment by lvca** [comment]
Created issue: https://github.com/ArcadeData/arcadedb/issues/624

**Reply by lvca** [reply]
I totally agree. Actually, I don't see the need to dump it into the console at all. From now on it will be dumped only to the configured log. It's already in the main branch.
